hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1780,6 +1791,7 @@ func reconcileCAPIManagerDeployment(deployment *appsv1.Deployment, hc *hyperv1.H
 						Args: []string{"--namespace", "$(MY_NAMESPACE)",
 							"--alsologtostderr",
 							"--v=4",
+							"--leader-elect=true",
```

Comment:
we don't want leader election right?

Reply:
yes we do for capi and capa only want one controller to be operating at a time.

Reply:
i'm dumb thank you thank you